content/metadata.yaml

```diff
@@ -261,6 +287,14 @@ authors:
         contributions:
           - Writing - Review & Editing
           - Visualization
+      vaccines-trad:
+        order: 3
+        contributions:
+          - Project Administration
```

Comment:
Did Nils contribute to the writing for either manuscript? I see some lines edited in your summary table.